app/models/concerns/post.rb

```diff
@@ -0,0 +1,61 @@
+module Post
+  extend ActiveSupport::Concern
+
+  include NameFromTitle
+  include Slug
```

Comment:
Maybe `Nameable` and `Sluggable` are closer to the actual convention, but then `Post` doesn't quite fit.